LoopKit/InsulinKit/ExponentialInsulinModel.swift

```diff
@@ -72,6 +72,12 @@ extension ExponentialInsulinModel: CustomDebugStringConvertible {
     }
 }
 
+extension ExponentialInsulinModel: Equatable {
+    public static func ==(lhs: ExponentialInsulinModel, rhs: ExponentialInsulinModel) -> Bool {
+        return abs(lhs.actionDuration - rhs.actionDuration) < .ulpOfOne && abs(lhs.peakActivityTime - rhs.peakActivityTime) < .ulpOfOne && abs(lhs.delay - rhs.delay) < .ulpOfOne
+    }
+}
```

Comment:
You've done well to follow precedent here in comparing to `ulpOfOne`, though in practice `ulpOfOne` isn't a great epsilon. In general, two ExponentialInsulinModel instances will either be created with the exact same values, or very different ones, and the values are static. You can safely adopt the auto-generated Equatable conformance here.